app/src/main/java/com/msg/sms_android/modules/DataSourceModule.kt

```diff
@@ -0,0 +1,17 @@
+package com.msg.sms_android.modules
+
+import com.msg.sms.data.remote.datasource.auth.AuthDataSource
+import com.msg.sms.data.remote.datasource.auth.AuthDataSourceImpl
+import dagger.Binds
+import dagger.Module
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+
+@Module
+@InstallIn(SingletonComponent::class)
+abstract class DataSourceModule {
+    @Binds
+    abstract fun provideAuthDataSource(
+        authDataSourceImpl: AuthDataSourceImpl
+    ): AuthDataSource
```

Comment:
```suggestion
    ) : AuthDataSource
```
이쪽부분 컨벤션이 혼자 동떨어진거같아요

Reply:
아 잘못봤네요